### DISCUSSION TITLE
GitHub Enterprise Server - VMware ESXi v8

### DISCUSSION THREAD

**Discussion by rhoenenergie-scholzno** [community/community#123616] [Enterprise] [answered]
Is it possible to install GitHub Enterprise Server on a ESXi v8 on VMware vSphere or is < v7 mandatory necessary?

**Comment by topeogunleye** [comment]
Hi Mr Noah, 

If you don't mind me asking, I'm new to GitHub Enterprise Server, how do I learn how to troubleshoot GitHub Enterprise Server? For Example:

1. I want to help users who have issues adding teams to a repo programmatically.
2. I want to help users troubleshoot Invalid SSL Cert errors, and lastly 
I want to help users troubleshoot performance issues on GitHub server.

**Comment by djdefi** [comment, marked as answer]
While we don’t actively block installations based on ESX version, as per our [install GitHub Enterprise Server on VMWare guide](https://docs.github.com/en/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-vmware#prerequisites), ESX versions 5.5 to 7.0 are currently tested and officially supported.

We anticipate officially testing and supporting version 8.0 in the future!
